.claude/requirements/done/REQ-002.md

```
@@ -0,0 +1,21 @@
+# [REQ-002] 시즌 주기 변경
+
+## 개요
+기존 1주 단위 시즌을 4개월 단위로 변경하여 유저가 상위 티어에 도달할 수 있는 충분한 기간을 확보한다.
+
+## API
+| Method | Path | 설명 |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)